### PR TITLE
Have TPC Neve independent

### LINKED ISSUE #154: Remove Neve dependency from TPC
### Description:
Remove Neve dependency from Neve to allow TPC to work as a stand-alone.
1. We would keep the current requirements of Template Cloud (it will require Neve Pro) to give access to saving and managing user templates
2. We would allow Starter Sites to install Neve if the theme is not present.

### PR DESCRIPTION
**Pull request by preda-bogdan**
### Summary

- [x] Remove the Notice that Neve is required from Plugins
- [x] I will remove the pop-up when Neve is not present on Starter Sites and add Neve as part of the required Install.
- [x] For Page Templates I will let them import the template without requiring them to have Neve.
- [x] I will ensure that if the Neve Pro license is available, the Template Cloud is available even if the Neve theme is not being used. My Library should be available on other themes if the License is active.
- [x] Show Upsell modal for Page Templates if Neve is not present
- [x] Change Upsell modal for Cloud Library if Neve is not present
- [x] Update Import modal, add a separate section for Theme to make it more obvious that the theme will be replaced.    

### Screenshots
Import Modal when Neve is not installed:
image

Upsell modal when Neve is not present for Page Templates or Cloud Library:
image

Upsell modal if Neve is present for Cloud Library:
image

### Test instructions

1. Check that TPC works as before with Neve and Neve Pro installed
2. Check that without Neve, the Starter Sites will install Neve and present that on Import
3. Check that without Neve, the Page Templates are not accessible, an Upsell is presented instead
4. Check that without Neve, the Cloud Library has a modified Upsell
5. Check that if you had Neve and Neve Pro active, for Neve Pro use this PR: https://github.com/Codeinwp/neve-pro-addon/pull/2087, see that Cloud Library is available if changing themes.

Closes #154.

### CODE REVIEW

**Comment by ineagu** [comment]
As I remember page templates aren't working great without Neve. so I am thinking that while we can leave those things there, if a user doesn't have Neve installed, I would hide starter sites, page templates & so on and focus on My templates part, with a nice onboarding screen/exp.

So I would look at the project as 2 sub-projects: 1. Pre-made templates and 2. Cloud Library.

**Comment by preda-bogdan** [comment]
Right now with the changes from here if the starter site or parts of the starter site will be imported, we will install and activate Neve for those. For the Page Templates, we do not require that. However, I agree that those should display correctly. I think the next step would be to review the Page Templates and make sure that they work independently of the installed theme.

The end goal is to allow the usage of Pre-made templates and Cloud Library without requiring the presence of Neve from my understanding. But I think it will happen in a couple of steps.

File: assets/src/Components/CloudLibrary/Library.js [hunk 1]
```diff
@@ -335,6 +333,41 @@ const Library = ( {
...
+		return (
+			<UpsellModal
+				title={ __( 'Coming soon' ) }
+				description={ sprintf(
+					// translators: %1$s: Theme Name %2$s Plugin Name.
+					__(
+						'Right now this feature is not available with your current setup. if you want to use it, you need to install %1$s theme and %2$s plugin',
+						'template-patterns-collection'
+					),
+					`<a href="${ themeURL }" target="_blank" rel="noreferrer">${ __(
```
Comment (cristian-ungureanu) [inline assets/src/Components/CloudLibrary/Library.js:348]: We should not translate theme and plugin name so no need to add __ before "Neve" and "Neve Pro Addon"

**Review by cristian-ungureanu — CHANGES_REQUESTED** [review]
Everything looks good @preda-bogdan, just one small change is required. You did a wonderful job 🚀

**Review by cristian-ungureanu — APPROVED** [review]

**Pull request merged**